index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Portfolio Homepage</title>
+</head>
+
+<body>
+  <div id="toolbar">
+    <div id="logoPlaceholder">
+      <p id="logoText">Welcome</p>
+    </div>

```

Comment:
Please make use of an `a` tag to wrap your logo. This is because the logo must always be a link that routes to the homepage when clicked. 

- You can read more about it [here](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) if you wish to learn about why is it standard for a website logo to navigate to the home page